x-pack/plugins/cases/server/client/utils.test.ts

```diff
@@ -489,4 +489,180 @@ describe('utils', () => {
       `);
     });
   });
+
+  describe('arraysDifference', () => {
+    it('returns null if originalValue is null', () => {
+      expect(arraysDifference(null, [])).toBeNull();
+    });
+
+    it('returns null if originalValue is undefined', () => {
+      expect(arraysDifference(undefined, [])).toBeNull();
+    });
+
+    it('returns null if originalValue is not an array', () => {
+      expect(arraysDifference('a string' as unknown as string[], [])).toBeNull();
```

Comment:
nit: It is perfectly fine to use `@ts-expect-error` on tests. This way you can avoid casting. A benefit of using `ts-expect-error` is that if there’s no error TypeScript will report that `ts-expect-error` wasn’t necessary.